Userland/Services/Taskbar/QuickLaunchWidget.cpp

```diff
@@ -23,6 +24,16 @@ QuickLaunchWidget::QuickLaunchWidget()
     set_frame_thickness(0);
     set_fixed_height(24);
 
+    m_context_menu = GUI::Menu::construct();
+    m_context_menu_default_action = GUI::Action::create("&Remove", [this](auto&) {
+        Config::remove_key("Taskbar", quick_launch, m_context_menu_app_name);
+        auto button = find_child_of_type_named<GUI::Button>(m_context_menu_app_name);
+        if (button) {
+            remove_child(*button);
+        }
```

Comment:
Sanity check: This is just to make it disappear faster, right? Because `QuickLaunchWidget::config_key_was_removed` will also attempt to remove it.

Reply:
Funny that you mention it, thought about that too. Well yes, but it takes time until that happens. The notifications are not send to the ConfigServer connection that issued the removal. We could simplify this by adding some kind of "notify_self" bool option to ConfigServer connection, but not sure if thats reasonable :^)

Reply:
It would be reasonable, but that's an entirely new mechanism and out of scope for this PR. Let's get this in (I think it can be merged as-is), and if you feel like it you could implement that mechanism.